src/routes/safe/store/actions/createTransaction.js

```diff
@@ -82,10 +56,9 @@ const createTransaction = ({
 
   const from = userAccountSelector(state)
   const safeInstance = await getGnosisSafeInstanceAt(safeAddress)
-  const threshold = await safeInstance.getThreshold()
   const lastTx = await getLastTx(safeAddress)
   const nonce = await getNewTxNonce(txNonce, lastTx, safeAddress)
-  const isExecution = (lastTx && lastTx.isExecuted && threshold.toNumber() === 1) || shouldExecute
+  const shouldAutomaticallyExecuteTx = await shouldAutomaticallyExecuteTransaction(safeInstance, nonce, lastTx)
```

Comment:
I think it can be shortened to `shouldExecuteTx` or even the old `isExecution` name - why didn't you like this one?
`shouldAutomaticallyExecuteTransaction` can be just `shouldExecuteTransaction` too

Reply:
Yes, I agree that it's a longvaria variable name. I opted for a long explanatory name to leave outside any misconception.

Originally the name was `shouldExecuteTransaction`, but I added the _automatically_ to make explicit what the function was trying to evaluate.

Anyways, explanation aside, if you guys are ok I'll rename it.

Reply:
what does automatically mean? we don't do anything automatically. we can either approve a transaction or execute it

Reply:
Yes, you're right, it's unnecessary.

Will change it as soon as I can.